### ISSUE TITLE
A workaroud as 1720 * 720 (hidpi) does not work.

### ISSUE THREAD

**Issue by LebronRice** [usr-sse2/RDM#23]
In edit, set add a new resolution 3840 * 1608 and click HiDpi, then save it and restart. You will see a new option: 1920 * 804 (hidpi) which is closest to 1720 * 720 (hidpi).

**Comment by jnooree** [comment]
Yes, because it is the twice of it. You need to add 3440 * 1440 for 1720 * 720 resolution.
